src/wast-parser.cc

```diff
@@ -1488,6 +1497,96 @@ Result WastParser::ParsePlainInstr(std::unique_ptr<Expr>* out_expr) {
   return Result::Ok;
 }
 
+// Simd
+// The expected V128 const lists is:
+// i32 0xXXXXXXXX 0xXXXXXXXX 0xXXXXXXXX 0xXXXXXXXX
+Result WastParser::ParseSimdConstList(ConstVector* consts) {
```

Comment:
Probably better if this had a different name; currently "ParseConstList" name is used to parse a list of constant values for assertions, like `(i32.const 1) (f32.const 2)` etc.

This only seems to be used by `ParseSimdConst` below, maybe it can just be inlined? Or do you think you'll need it elsewhere?

Reply:
Inline it will cause ParseSimdConst Function bigger.  Let me try it.